submissions/kolawoleOgunfowokan/HTML/index.html

```diff
@@ -197,19 +197,83 @@
             </div>
```

Comment:
This seems a bit off of what is required in the task. The task states that you should render products on the page with Javascript. You seem to already have products on this page.

Reply:
I have fixed that, I actually place them there as a placeholder. I have removed them now, and they displaying well.

Reply:
This approach is not efficient because if you add 100 more objects to your product array, then you have to add 100 more placeholders on your page. Try to make it such that the products are dynamically generated on the page

Reply:
I have worked on the files and now the products are dynamically generated.

Reply:
I have worked on it. Thank you.